wiki/content/clients/index.md

```diff
@@ -439,6 +439,14 @@ fmt.Println(string(resp.Json))
 
 ```
 
+## C\#
+
+The official C# client [can be found here](https://github.com/dgraph-io/dgraph.net).
+Follow the instructions in the [README](https://github.com/dgraph-io/dgraph.net#readme) to get
+it up and running.
+
+More details on the supported versions can be found at
+[this link](https://github.com/dgraph-io/Dgraph-dotnet#supported-versions).
```

Comment:
this one needs to be changed as well. New link: https://github.com/dgraph-io/dgraph.net#supported-versions

Reply:
Thanks, @sleto-it.